pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -55,12 +55,12 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 		if name == lastPlanStatus.Name {
 			planDisplay := fmt.Sprintf("Plan %s (%s strategy) [%s]%s", name, plan.Strategy, lastPlanStatus.Status, printMessageIfAvailable(lastPlanStatus.Message))
 			planBranchName := rootBranchName.AddBranch(planDisplay)
-			for _, phase := range lastPlanStatus.Phases {
-				phaseDisplay := fmt.Sprintf("Phase %s [%s]%s", phase.Name, phase.Status, printMessageIfAvailable(phase.Message))
+			for i, lastPhase := range lastPlanStatus.Phases {
```

Comment:
I was all the time wondering what the last mean here, maybe we should call this just phaseStatus/stepStatus?

Reply:
I've reverted to the original naming here.